### DISCUSSION TITLE
Automatically stop playback when bluetooth headphones are disconnected?

### DISCUSSION THREAD

**Discussion by ladeve** [KRTirtho/spotube#300] [General]
Hi, 
Should the Android application automatically pause the playback when bluetooth headphones are disconnected or is it a bug that the music just continues to play from the phone speakers?

**Comment by mfyddti** [comment]
Brother, are you Android?

**Reply by ladeve** [reply]
Yes

**Comment by Rafee-M** [comment]
It's usually the default behavior of Android to disable playback on Bluetooth device disconnect.
